### ISSUE TITLE
No PDF on Windows 10

### ISSUE THREAD

**Issue by haukefrahmann** [thawn/ttmp32gme#8]
Says "no such file ... "

"watching" the folder during process >> no pdf ist created

also, there is no "print button", only "Save as Pdf" >> is that right?

**Comment by thawn** [comment]
Printing under windows 10 only works using the Edge browser. There you can print directly from the browser.

On my win 10 test vm, pdf generation works.

**Comment by haukefrahmann** [comment]
Did in Edge, did not work

screenshot added

Am 25.03.2017 um 20:41 schrieb thawn:

**Comment by thawn** [comment]
did you try to use the print function of edge?

**Comment by haukefrahmann** [comment]
very strange, print does nothing >> no print job generated ...


Am 25.03.2017 um 21:09 schrieb thawn:

**Comment by thawn** [comment]
did you try ctrl + p ?

**Comment by haukefrahmann** [comment]
ähm... i used the printing function and *after *sending to the printer 
it did nothing
normal website print does the normal printing thing.

very strange, as I said...

Am 26.03.2017 um 11:11 schrieb thawn:

**Comment by thawn** [comment]
did you select any album before you clicked on "print selected" in the 
library?

Do you see a preview of the album on the print page?

please upload a screenshot of the command window that opens with the 
program and of the print page.

you will need to follow these instructions (attachments to emails don't 
work):

https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/

Cheers

Till

**Comment by mattanja** [comment]
Hi, can confirm the issue on Win10. I think I may have identified the source of the problem:
```
C:\Users\matta\AppData\Local\Temp\par-6d61747461\cache-d7d821b0c530d69e96d9057e179ba6e61448b6c6\inc\lib\wkhtmltopdf.exe -B 0.5in -T 0.5in -L 0.5in -R 0.5in http://localhost:10020/pdf C:\Users\matta\AppData\Roaming\ttmp32gme\library\print.pdf
Qt: Untested Windows version 10.0 detected!
Loading page (1/2)
unhandled callback exception on event (/print.pdf, AnyEvent::HTTPD=HASH(0x9362500), AnyEvent::HTTPD::Request=HASH(0x938cf08)): Can't read C:\Users\matta\AppData\Roaming\ttmp32gme\library\print.pdf: No such file or directory at TTMp32Gme/LibraryHandler.pm line 173.
```

Running wkhtmltopdf.exe in the tmp folder shows this dialog:
![image](https://cloud.githubusercontent.com/assets/654416/25070262/3a91534a-229a-11e7-9318-b89fc85a6a50.png)
Next this
![image](https://cloud.githubusercontent.com/assets/654416/25070266/471dbe6e-229a-11e7-9ab0-09d8ece612a8.png)

Maybe this thread helps?
http://stackoverflow.com/questions/6404636/libstdc-6-dll-not-found

Thank you for your efforts!

**Comment by thawn** [comment]
@mattanja thanks for the additional information.
This is very strange since I cannot reproduce it in my test VM, which is a fresh install of win10 professional.

It seems to be an issue with some libraries missing on your system. Ineed, I do not bundle these libraries because I found that they were not needed on my test vm.

I suspect, that this may be related to the Windows Subsystem for Linux providing the necessary libraries on my test system.

I will need some more help to try and track down or fix this issue.
Could you please go to to Settings>System>About and tell me your OS Build as well as the system type (32 bit or 64 bit)?

Could you please test the new version of ttmp32gme you can download here (link expires on April 30):
https://thawn.noip.me/owncloud/index.php/s/2KoYqOkgjEPCmLL

**Comment by thawn** [comment]
can you also please check, whether the file libstdc++-6.dll which it is complaining about is present in the temporary folder where wkhtml2pdf is executed?

on my test system it is present in the temp folder so I suspect that the libraries were bundled automatically as they should have been. Maybe you are running a 32 bit system?

**Comment by thawn** [comment]
closing due to inactivity. If you can provide the required information, please reopen the issue.

**Comment by thawn** [comment]
O.k. I just found the problem. The issue only occurred when there was no cover image.
The problem is fixed in v 0.2.1
